src/Controls/src/Core/Application.cs

```diff
@@ -437,5 +437,7 @@ protected internal virtual void CleanUp()
 
 			NavigationProxy = null;
 		}
+
+		IReadOnlyList<Maui.IElement> IVisualTreeHelper.GetVisualChildren() => new List<Maui.IElement> { MainPage }.AsReadOnly();
```

Comment:
This should probably return the windows, and then those windows will have a Page/Content property to get this. We could totally have a window that wants to hot reload a title

Reply:
Updated, it needs to return Windows or else it won't work correctly.